draft-ietf-quic-transport.md

```diff
@@ -3804,6 +3804,14 @@ number length, connection ID length, and path MTU.  A receiver MUST accept
 packets containing an outdated frame, such as a MAX_DATA frame carrying a
 smaller maximum data than one found in an older packet.
 
+When a sender declares a packet as lost and either retransmits the contained
+frames or marks them for retransmission, the sender SHOULD avoid subsequent
+retransmission of this information if it later receives an acknowledgement for
+that packet. Doing so requires senders to retain information about packets after
+they are declared lost. A sender can discard this information after a period of
+time elapses, such as three times the PTO (Section 6.2 of {{QUIC-RECOVERY}}), or
+on other events, such as reaching a memory limit.
```

Comment:
```suggestion
Acknowledgments can be received after a sender declares a packet as lost.  A
sender SHOULD avoid sending information again after receiving a late
acknowledgment. Doing so requires senders to retain information about packets
for some time after they are declared lost. A sender can discard this
information after a period of time elapses, such as three times the PTO (Section
6.2 of {{QUIC-RECOVERY}}), or on other events, such as reaching a memory limit.
```

Reply:
I think @martinthomson's idea of splitting the sentence into two. The only thing I might prefer is changing the last part of the second sentence "after receiving a late acknowledgement" to something like "after receiving such an acknowledgement" to make it clearer, but anyways it's good.

Reply:
How about "..after receiving an acknowledgement for a previously lost packet."  I think late acknowledgement is potentially confusing.

Reply:
And how about starting with "A packet can be acknowledged after being declared lost." instead of "Acknowledgements can be received after a sender declares a packet as lost."